src/main/java/uk/gov/hmcts/sptribs/notification/NotificationServiceCIC.java

```diff
@@ -36,30 +36,44 @@
 public class NotificationServiceCIC {
 
     @Autowired
-    private NotificationClient notificationClient;
+    private final NotificationClient notificationClient;
 
     @Autowired
-    private EmailTemplatesConfigCIC emailTemplatesConfig;
+    private final EmailTemplatesConfigCIC emailTemplatesConfig;
 
     @Autowired
-    private IdamService idamService;
+    private final IdamService idamService;
 
```

Comment:
We have another notification module under common module. If we don't have any JIRA ticket to cover that, feel free to raise one.

Reply:
Maybe a task can be added to this ticket to remove the common > notification package and place everything inside the notification module ?
[https://tools.hmcts.net/jira/browse/DTSSTCI-318](url)